test/linqmacro.jl

```diff
@@ -70,6 +68,7 @@ xlinq3 = @linq df  |>
         transform(cols(y_str) = 10 * cols(x_sym))  |>
         DataFrames.groupby(b_str) |>
         orderby(-mean(cols(x_sym)))  |>
+        groupby(:b) |>
         based_on(cols("meanX") = mean(:x), meanY = mean(:y))
```

Comment:
This example is really weird. Why isn't `orderby` called as the last step?

Reply:
Thanks, I have changed this pipe to match the other ones (it should be the same, just adding `cols` everywhere).